index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, minimum-scale=1.0">
+    <title>PORTFOLIO</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header>
+       <div class="logo"><img src="images/image_geometry_1.png" alt="logo image"></div>
```

Comment:
- [ ] Since this is a logo, it should use a tag and you will be able to add href so that upon clicking it, it can always go to the home page which is the index page. Generally, it is for a convention, semantics, and user experience(UX) reasons.